src/test/java/com/patrycja/pound/services/DogServiceTest.java

```diff
@@ -0,0 +1,97 @@
+package com.patrycja.pound.services;
+
+import com.patrycja.pound.models.domain.Dog;
+import com.patrycja.pound.models.domain.Zookeeper;
+import com.patrycja.pound.models.dto.DogDTO;
+import com.patrycja.pound.repository.DogRepository;
+import com.patrycja.pound.services.mappers.DogMapper;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DogServiceTest {
+
+    @Mock
+    DogRepository dogRepository;
+    @Mock
+    DogMapper dogMapper;
+    @Mock
+    ZookeeperService zookeeperService;
+
+    @InjectMocks
+    DogService dogService;
```

Comment:
Why all fields are not private?
Can we also switch `@InjectMocks` with constructor in set up method?